QuasarHelios2015/src/org/team1540/quasarhelios/Elevator.java

```diff
@@ -64,8 +64,8 @@ public void event() {
     public static final EventOutput setBottom = EventMixing.combine(raising.getSetFalseEvent(), lowering.getSetTrueEvent());
     public static final EventOutput stop = BooleanMixing.getSetEvent(BooleanMixing.combine(raising, lowering), false);
 
-    private static final BooleanStatus atTopStatus = new BooleanStatus();
-    private static final BooleanStatus atBottomStatus = new BooleanStatus();
+    public static final BooleanStatus atTopStatus = new BooleanStatus();
+    public static final BooleanStatus atBottomStatus = new BooleanStatus();
```

Comment:
It's so it can be read from AutoLoader/Ejector. In general, however, it's nice to know where the elevator is at any given time.